README.md

```diff
@@ -7,27 +7,27 @@ This repository is for standards development of Interchain Standards. Initially
 #### Stage 0 - `Strawman`
 - _**Purpose**_: Start the specification process
 - _**Entrance Criteria**_: [Open an issue](https://github.com/cosmos/ics/issues/new) on this repository with a short outline of your proposal
```

Comment:
Maybe also add here that issue must have a spec-name and spec-number? Or do we assign those later?

Reply:
Let's require a name but not a number